public/styles/mermaid.css

```
@@ -0,0 +1,45 @@
+	--mermaid-primary-color: var(--primary20);
+	--mermaid-secondary-color: var(--primary30);
+	--mermaid-tertiary-color: var(--primary40);
+}
+
+pre.mermaid {
+	margin: var(--site-spacing) 0;
+	padding: var(--spc-4x);
+	overflow-x: auto;
+	text-align: center;
+
+	background-color: var(--background_primary);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)